chef/cookbooks/ceph/recipes/radosgw_ha.rb

```diff
@@ -31,18 +31,30 @@
 # Avoid races when creating pacemaker resources
 crowbar_pacemaker_sync_mark "wait-ceph-radosgw_ha_resources"
 
-service_name = "ceph-radosgw"
+transaction_objects = []
 
+service_name = "ceph-radosgw"
 pacemaker_primitive service_name do
   agent node[:ceph][:ha][:radosgw][:agent]
   op node[:ceph][:ha][:radosgw][:op]
-  action :create
+  action :update
   only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
 end
+transaction_objects << "pacemaker_primitive[#{service_name}]"
```

Comment:
Why don't want to use `<<` operator? Codeclimate and hound are OK with this code.